features/step_definitions/context.rb

```diff
@@ -67,3 +67,11 @@
   visit getURI(item)
 end
 
+Soit("le point de vue {string} est en mode édition") do |viewpoint|
+  click_on class: 'creationButton'
+  visit '/viewpoint/' + getUUID('')
```

Comment:
I'm not sure about the necessity of both clicking the button and then visiting the page...